mandatory/1-syntax-errors.js

```diff
@@ -1,18 +1,25 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+//addNumbers adds numbers correctly
+function addNumbers(a, b, c) {
   return a + b + c;
 }
+console.log(addNumbers(3, 4, 6));
```

Comment:
In the test, we don't want to leave any debugging content.
All the `console.log`s are for debuggying purposes only. You should remove them from test files.
The tests will call the functions at the end of the file and check the results.